operations/helm/charts/mimir-distributed/templates/provisioner/provisioner-job.yaml

```diff
@@ -115,4 +117,10 @@ spec:
             secretName: {{ .Values.tokengenJob.adminTokenSecret }}
         - name: bootstrap
           emptyDir: {}
-{{- end -}} 
\ No newline at end of file
+        {{- if .Values.provisioner.extraVolumes }}
```

Comment:
can you document this in the values.yaml, please?

Reply:
done